db/migrate/20220810000929_add_user_ref_to_post.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToPost < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :users, null: false, foreign_key: true
```

Comment:
* Kindly stick to the information provided in the ERD diagram as i mentioned above to create your foreign keys